include/common.sh

```diff
@@ -170,7 +170,18 @@ writeConfigFile(){
 # Main root location for the zopen installation; can be changed if the
 # underlying root location is copied/moved elsewhere as locations are
 # relative to this envvar value
-ZOPEN_ROOTFS="${rootfs}"
+knv=false
+exportknv=""
+if [ \$# -gt 0 ]; then
```

Comment:
Wondering if we need a --help to list out the supported options for zopen-config

Reply:
> Wondering if we need a --help to list out the supported options for zopen-config
If we plan to add more capability to this script then it probably makes sense to, and perhaps to prefix them with '--' for consistency with other scripts.

Reply:
Note, as this script is designed to be sourced into a users environment NOT run directly, adding in a blocking hashbang.

Reply:
I wonder if it makes sense to allow it be run in the case of --knv or --eknv since it's not altering the environment. But I think without the bash req, it's difficult to determine that so I'm ok with this.